handlers/attachment_resource.go

```diff
@@ -27,6 +27,21 @@ func HandleSubmitAttachment(svc dao.Service) http.Handler {
 
 		log.InfoR(req, fmt.Sprintf("start POST request for submit attachment with transaction id: %s", transactionID))
 
+		// Check if transaction is closed
```

Comment:
If this block of code is reused throughout the PR, shouldn't it be refactored into a func and reused?